admin_manual/configuration/server/occ_command.rst

```diff
@@ -1238,7 +1263,7 @@ Instead, you need to use the ``config:app:set`` command, as in the following exa
 
 ::
    
-  occ config:app:set user_ldap updateAttributesInterval --value=7200
+  sudo -u www-data php occ config:app:set user_ldap updateAttributesInterval --value=7200
```

Comment:
I guess since the other examples have the ``sudo`` stuff at the front, we should be consistent.
Of course, it depends exactly how you have installed ownCloud as to what ``sudo`` or other stuff might need to go in front of the actual ``occ`` command.

Reply:
Yes, a matter of consistency